lib/comment/comment.routes.js

```diff
@@ -0,0 +1,47 @@
+const Joi = require('joi');
+
+
+module.exports = {
+  routes: (models) => {
+    // const controllers = require('./comment.controllers')(models);
+    const commentModels = require('./comment.models');
+
+    return [
+        {
+            method: 'GET',
+            path: '/comments',
+            // handler: controllers.viewAllComments,
+            // TODO: move this into controllers file and expand into real logic
+            handler: (request, h) => { return {param: request.params.id,}},
+            config: {
+              auth: 'jwt',
+              description: 'Get Comments',
+              notes: 'Returns all comments.',
+              tags: ['api', 'Comment'],
+              validate: {
+                query: commentModels.apiFilterQuery,
```

Comment:
apiFilterQuery isn't defined. You could remove this for now as it would probably only be needed if you were creating a comment searching functionality.